### PR TITLE
WIP: Bitbucket Guide

### PR DESCRIPTION
**Pull request by sukottokun**
Bitbucket + CircleCI, using the example project. @alexfornuto @greg-1-anderson if I can get a review por favor.

### CODE REVIEW

File: source/_docs/guides/bitbucket.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+
+## Step 1: Create the Example Project and Add Version Control
+
+1. From your terminal, use composer to make a new local project, called "bitbucket," based on our example repo:
+
+```nohighlight
+composer create-project pantheon-systems/example-drops-8-composer bitbucket
+```
+
+2. Important Note: Currently, a plugin used to manage multidev environments is only configured to work with GitHub. While a fix will be released shortly in a future version, tests will fail because of this build step. At this point, now would be a good time to comment out the following line in the ```bitbucket/circle.yml``` file we just cloned:
```
Comment (greg-1-anderson) [inline source/_docs/guides/bitbucket.md]: We should either:

1. Hold off on publishing this guide until the build_env:delete commands work with bitbucket.
 - or -
2. Write a bash script that does the equivalent of build:env:delete:ci for Bitbucket, and make a Bitbucket version of example-drops-8-composer that contains this script.

Comment (sukottokun) [inline source/_docs/guides/bitbucket.md]: Thanks for the feedback @greg-1-anderson! What is the timeline on #1?
Regarding #2, Are you in any position to work together on this? Your help would probably be the difference in doing this in a day or a week or more if I do it myself (I'm guessing). I can work on your time one day this week.

File: source/_docs/guides/bitbucket.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+
+1. Create a new Bitbucket repository.
+
+![Bitbucket Dashboard](/source/docs/assets/images/integrations/bitbucket/bb_repo.png)
+
+
+2. Next, follow the instructions Bitbucket presents to push the existing code to the new remote repository:
+
+```nohighlight
+git remote add origin https://sukottokun@bitbucket.org/sukottokun/drupal-site.git
```
Comment (greg-1-anderson) [inline source/_docs/guides/bitbucket.md]: Maybe use a generic org or user in examples?

File: source/_docs/guides/bitbucket.md [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+Now you can rerun the failed job again from the Builds tab in CircleCI and you can follow the build from the dashboard. This time it should pass. 
+
+![Running tests](/source/docs/assets/images/integrations/bitbucket/running.png)
+
+## Final Step: Test the New Workflow
+Now try to submit a pull request. Let's add something to the repo and test the workflow. From the terminal:
+
+```
+git checkout -b test
+vim tests/features/content-ui.feature
```
Comment (greg-1-anderson) [inline source/_docs/guides/bitbucket.md]: Should be consistent with other guides vis-a-vis instructions on editing files. Probably best to avoid vim.

Comment (alexfornuto) [inline source/_docs/guides/bitbucket.md]: I'll handle style issues like that during the copy edit, don't worry about it for now. So long as the tech is correct, I'll make it pretty later 😉.

**Comment by alexfornuto** [comment]
@sukottokun sounds like you and @greg-1-anderson are discussing some possible changes on or around L39. Please give me a buzz once the "technical review" there is addressed, then I'll dive in.

File: source/_docs/guides/bitbucket.md (hunk 1 above)
Comment (greg-1-anderson) [inline source/_docs/guides/bitbucket.md]: Early in the week I need to work on my objective for this sprint. I might be able to work with you on #2 later in the week. If not, I'll talk with @ari-gold about adding time for #1 or #2 to next sprint.

Comment (greg-1-anderson) [inline source/_docs/guides/bitbucket.md]: c.f. https://developer.atlassian.com/bitbucket/api/2/reference/resource/repositories/%7Busername%7D/%7Brepo_slug%7D/pullrequests

Comment (sukottokun) [inline source/_docs/guides/bitbucket.md]: Great. Let me know about this week, I can work around your schedule @greg-1-anderson !

**Comment by alexfornuto** [comment]
@sukottokun @greg-1-anderson do either of you have any updates, or is this ready for review?

**Comment by sukottokun** [comment]
@alexfornuto No, still working on it.

**Comment by rachelwhitton** [comment]
I think the long term goal is to support additional providers in the example projects or in the build tools plugin. Closing this for now since it seems to have gone stale - feel free to reopen if that's not the case.

**Pull request closed without merging**